packages/application-components/src/components/project-stamp/project-stamp.tsx

```diff
@@ -0,0 +1,71 @@
+import { ReactElement } from 'react';
+import { css } from '@emotion/react';
+import { MessageDescriptor, useIntl } from 'react-intl';
+import { DotIcon } from '@commercetools-uikit/icons';
+import Stamp, { TTone } from '@commercetools-uikit/stamp';
+import messages from './messages';
+
+type TCustomStampProps = {
+  tone: TTone;
+  label: MessageDescriptor & { values?: Record<string, string | number> };
+  icon?: ReactElement;
+};
+function CustomStamp(props: TCustomStampProps) {
+  const intl = useIntl();
+  const { values, ...message } = props.label;
+  return (
+    <Stamp
+      tone={props.tone}
+      isCondensed={true}
+      label={intl.formatMessage(message, values || {})}
+      icon={props.icon}
+    />
+  );
+}
+
+const IsProduction = () => (
+  <CustomStamp
+    tone="positive"
+    label={messages.ProjectProduction}
+    icon={
+      <div
+        css={css`
+          height: 18px;
+          svg {
+            height: 18px;
+            width: 12px;
+          }
+        `}
+      >
+        <DotIcon color="primary" />
+      </div>
+    }
+  />
+);
+
+const IsSuspended = () => (
+  <CustomStamp tone="critical" label={messages.ProjectSuspended} />
+);
+
+const IsExpired = () => (
+  <CustomStamp tone="critical" label={messages.ProjectExpired} />
+);
+
+const WillExpire = (props: { daysLeft: number }) => (
+  <CustomStamp
+    tone="critical"
```

Comment:
I agree as well, that it might be better if it's `information` tone. [444bfd3](https://github.com/commercetools/merchant-center-application-kit/pull/3403/commits/444bfd31f07daf4ef8411d9a80f34f774dd548e8)